setup.py

```diff
@@ -0,0 +1,34 @@
+from pathlib import Path
+
+from setuptools import find_packages, setup
+
+_version_file_path = Path(__file__).parent.resolve() / ".version"
+with _version_file_path.open("r") as file:
+    _version = file.read().strip()
+
+
+setup(
+    name="bw-check-commit-hook",
```

Comment:
In general we want to use BWT instead of BW, and the repo/package should be `check-commit-hook` (`checkpatch-hook` would be better really...). This applies everywhere, including directory names, i.e. change the dir name to `check_commit_hook`.